docs/ops/comparison/IsNaN_10.md

```diff
@@ -0,0 +1,48 @@
+# IsNaN {#openvino_docs_ops_comparison_IsNaN_10}
+
+**Versioned name**: *IsNaN-10*
+
+**Category**: *Comparison*
+
+**Short description**: *IsNaN* returns the boolean mask of a given tensor which maps `NaN` to `True`.
+
+**Detailed description**: *IsNaN* returns the bolean mask of the input tensor in which `True` corresponds to `NaN` and `False` to other values.
+* The output tensor has the same shape as input tensor.
+* The `i`'th element of the output tensor is `True` if  `i`'th element of the input tensor is `NaN`. Otherwise it is `False`.
+* For example, for given input tensor `[NaN, 2.1, 3.7, NaN, Inf]` the output tensor is `[True, False, False, True, False]`.
```

Comment:
(Optional) Example values can be also added to the "Examples" section. 
Reference: [Select_1.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/condition/Select_1.md), [CumSum_3.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/CumSum_3.md )